sdk/attestation/service.projects

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup>
+    <!-- Remove data plane tests from nightly live runs: https://github.com/Azure/azure-sdk-for-net/issues/10437 -->
```

Comment:
This comment isn't applicable to your usage so I'd remove it or update it for your case.